doc-generator/src/main/scala/DocGenerator.scala

```diff
@@ -0,0 +1,95 @@
+import com.codacy.plugins.api._
+import com.codacy.plugins.api.results._
+import play.api.libs.json.Json
+import scala.xml._
+
+object DocGenerator extends App {
+  val scalastyleDefinition = XML.loadString(os.read(os.resource / "scalastyle_definition.xml"))
```

Comment:
where do those files come from? Have you added them to the repo, or come from the jar? Ideally we just read them from that jar that we already depend on, right?

Reply:
Yes, I'm reading them from the classpath resources.
Since I'm depending on the scalastyle jar I the classpath I have its resources, which contain the xml files https://github.com/beautiful-scala/scalastyle/blob/master/src/main/resources

Reply:
Then can we remove the https://github.com/codacy/codacy-scalastyle/blob/master/docs/scalastyle_config.xml ? Or what is the deal there? can we rely on "xml on resources", so people also don't assume that we are reading xmls on the repo?

Reply:
> Then can we remove the https://github.com/codacy/codacy-scalastyle/blob/master/docs/scalastyle_config.xml ? Or what is the deal there? can we rely on "xml on resources", so people also don't assume that we are reading xmls on the repo?

Didn't notice we were depending on that file. I changed the tool to use the default configuration from the Scalastyle lib, so we don't have to copy paste any file and we're safe if they rename it in the original repository